spatialprofilingtoolbox/workflow/dataset_designs/multiplexed_imaging/halo_cell_metadata_design.py

```diff
@@ -16,17 +13,19 @@ class HALOCellMetadataDesign:
     This class provides the schema necessary to interpret cell metadata manifests
     exported from the HALO software.
     """
+
     def __init__(self,
-        elementary_phenotypes_file: str=None,
-        compartments_file: str=None,
-        **kwargs,
-    ):
```

Comment:
Was this deliberate? Seems like the previous indenting method here was consistent with the rest of the code.

Reply:
Oh I think I see the logic, it's matching the indent of the first open parenthesis?

Reply:
Yes. If `self` was also on its own line instead of with the first open paren, it would follow the original indenting format. It's easy to forget that `self` is also an input arg.